src/dev-1/ThreeOption.java

```diff
@@ -0,0 +1,25 @@
+import java.util.ArrayList;
+
+/**
+ * creation ArrayList
+ */
+public class ThreeOption {
+    /**
+     * method returns a list of the required number of options
+     * @param args a list of all options
+     * @param numberOption the amount required output options
+     * @return return a list of the required number of options
+     * @throws Exception do put option
+     */
+    public ArrayList<String> getOption(String[] args, int numberOption) throws Exception {
+        ArrayList<String> options = new ArrayList<>();
+        if (args.length == 0) {
+            throw new Exception("you do not enter an option");
```

Comment:
class Exception is too generic to be used here - try to find more specific exception for invalid argument case